SignItCoreContent.js

```diff
@@ -1,49 +1,38 @@
-var SignItCoreContent = function (locale,mapi18n) {
-  const sourceMap = new Map(mapi18n);
-  banana = { i18n: (msg) => sourceMap.get(locale)[msg] };
-  console.log("Passed trough ! :", locale);
-  console.log("SignItCoreContent.js",banana );
-  // let hlwa = browser.i18n.getMessage("si-panel-videos-title"); 
-  // console.log("hlwa = ",hlwa); 
+var SignItCoreContent = function () {
+  console.log("SignItCoreContent.js");
```

Comment:
Why are we logging on console ?

Reply:
This is legacy, for when i inspected, to know to which file the following logs were related to. Ca be removed if needed